indexd_test_utils/__init__.py

```diff
@@ -140,7 +140,7 @@ def indexd_client(indexd_server, create_indexd_tables, indexd_admin_user):
 
 
 @pytest.fixture(scope="session")
-def indexd_server():
+def indexd_server() -> "MockServer":
```

Comment:
I belive class here is not supported in older version of python.

Reply:
Ah, understood. Well, in that case, is this providing additional value from a type hinting standpoint? Is this the standard way to handle this for older versions of Python? Just asking questions since I haven't seen this before (but just because I haven't seen it doesn't mean it's not the correct approach).

Reply:
Sorry, I made a mistake, updated it. 
Have to move MockServer before this fixture.

Reply:
No worries, thank you for looking into this and making the updates. Looks good!